Controller/Api/GalleryController.php

```diff
@@ -343,7 +343,7 @@ protected function handleWriteGalleryhasmedia(GalleryInterface $gallery, MediaIn
             'csrf_protection' => false,
         ));
 
-        $form->bind($request);
+        $form->handleRequest($request);
```

Comment:
That's the reason for a new PR. If we rollback this PR, the *improving the code* will also be reverted too.